x-pack/solutions/security/plugins/security_solution/common/api/entity_analytics/entity_store/common.schema.yaml

```diff
@@ -13,7 +13,13 @@ components:
         - host
         - service
         - generic
-
+    EntityECSField:
```

Comment:
nit: naming is hard, but this isn't very descriptive, do we need another word? Something like "base entity field" maybe? I agree it isnt the entity type of course

Reply:
Agreed with mark to go ahead with `BaseECSEntityField`